packages/file-type-icons/src/initializeFileTypeIcons.tsx

```diff
@@ -5,8 +5,8 @@ import { FileTypeIconMap } from './FileTypeIconMap';
 const PNG_SUFFIX = '_png';
 const SVG_SUFFIX = '_svg';
 
-const DEFAULT_BASE_URL = 'https://spoprod-a.akamaihd.net/files/fabric-cdn-prod_20201008.001/assets/item-types/';
-const ICON_SIZES: number[] = [16, 20, 24, 32, 40, 48, 64, 96];
+export const DEFAULT_BASE_URL = 'https://spoprod-a.akamaihd.net/files/fabric-cdn-prod_20201008.001/assets/item-types/';
```

Comment:
A little confused why these are being exported.

Reply:
(and the icon_sizes)

Reply:
I am importing both of these in my test file. Also I need the default url while emitting the HTML as string from the new API.